### PR TITLE
Bug/38

### PR DESCRIPTION
**Pull request by congocongo**
@anemitz 
for #38

### CODE REVIEW

**Comment by congocongo** [comment]
@anemitz can you review and accept?

File: scripts/bulk_update_leads_info.py [hunk 1]
```diff
@@ -56,12 +56,15 @@ def get_contact_info(contact_no, csv_row, what, contact_type):
...
 
-contact columns (new contacts wil be added):
+contact columns (new contacts will be added):
     * contact[0-9]_name                 - contact name
     * contact[0-9]_title                - contact title
     * contact[0-9]_phone[0-9]           - contact phones
     * contact[0-9]_email[0-9]           - contact emails
     * contact[0-9]_url[0-9]             - contact urls
+
+custom fields (new custom types will be created, new fields will be added):
```
Comment (philfreo) [inline scripts/bulk_update_leads_info.py]: what do you mean by "new custom types will be created"?

Comment (congocongo) [inline scripts/bulk_update_leads_info.py]: this part

https://github.com/congocongo/closeio-api/blob/master/scripts/bulk_update_leads_info.py#L97-L111

2015-04-01 1:27 GMT+07:00 Phil Freo <notifications@github.com>:

Comment (philfreo) [inline scripts/bulk_update_leads_info.py]: yeah but this english description doesn't make much sense, also you don't mention create_custom_fields. can you reword it to clarify it? imagine someone using this script that doesn't read the code

Comment (philfreo) [inline scripts/bulk_update_leads_info.py]: @congocongo can you improve this?
